package-system/zlib/build_zlib_mac.sh

```diff
@@ -6,15 +6,12 @@
 #
 #
 
-cmake -S temp/src -B temp/build -G Xcode -DCMAKE_CXX_STANDARD=17 -DSKIP_INSTALL_FILES=YES
-if [ $? -ne 0 ]; then
-    echo "Error generating build"
-    exit 1
-fi
+# note that toolchain path is relative to the source path (-S) not to the folder this script lives in.
+cmake -S temp/src -B temp/build -G Xcode \
+    -DCMAKE_CXX_STANDARD=17 \
+    -DSKIP_INSTALL_FILES=YES \
+    -DCMAKE_TOOLCHAIN_FILE=../../../../Scripts/cmake/Platform/Mac/Toolchain_mac.cmake || exit 1
+
+cmake --build temp/build --target zlibstatic --parallel --config Release -j 8 || exit 1
```

Comment:
the same parameter is being passed: `--parallel` and `-j` are both the same: https://cmake.org/cmake/help/latest/manual/cmake.1.html#build-tool-mode

Note that the native option for Ninja and XCode is to use all cores, so in those cases, it doesnt need to be passed.

Reply:
will fix